client/js/options.js

```diff
@@ -111,6 +138,84 @@ $("#desktopNotifications").on("change", function() {
 	}
 });
 
+function onPushButton() {
```

Comment:
Can we abstract this out into another module? It doesn't feel like it's anything actually to do with the options itself.